tests/test_modeling_common.py

```diff
@@ -4640,62 +4415,6 @@ def test_flash_attn_2_can_dispatch_composite_models(self):
                     if not has_fa2:
                         raise ValueError("The FA2 model should have FA2 layers")
 
-    @require_flash_attn
-    @require_torch_gpu
-    @mark.flash_attn_test
-    @slow
-    def test_flash_attn_2_generate_reuse_cache(self):
```

Comment:
Similar to the FA2 + padding tests above.

In this case, we check that we can reuse a cache in `test_generate_continue_from_past_key_values`, no need for a FA2-specific test. 

Deleted.